cluster.ml

```diff
@@ -70,6 +70,7 @@ let deployment =
       ~zone:gcloud_zone
       ~java:`Oracle_7
       ~machine_type:(`Google_cloud `Highmem_8)
+      ~additional_packages:["postgresql-9.4"]
```

Comment:
Not sure if it is and important problem but I think on debian/ubuntu, installing the PG package also starts the server.